app/src/main/java/com/ecosystem/kin/app/MainActivity.java

```diff
@@ -174,6 +174,11 @@ private void copyToClipboard(CharSequence textToCopy) {
 
     private void getBalance() {
         try {
+            //Get Cached Balance
+            int cachedBalance = Kin.getCachedBalance();
+            Log.d(TAG, "getBalance: Cached Balance is: " + cachedBalance);
```

Comment:
balanceView.setText(getString(R.string.get_balance_d, balance)); - just for seeing how it affect the UI

Reply:
👍